keps/sig-instrumentation/0034-distributed-tracing-kep.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Leveraging Distributed Tracing to Understand Kubernetes Object Lifecycles
+authors:
+  - "@Monkeyanator"
+editors:
+  - "@dashpole"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-architecture
+  - sig-node
```

Comment:
If this proposes changes to api calls (parameters, headers, etc) or api objects (storing new interesting things) then api machinery probably needs to be involved...

Reply:
It doesn't currently, as I plan to use annotations for the alpha stage, but it will if it moves beyond that stage.  I'll add api-machinery.